mkdocs/docs/reference/tasks/notification.md

```diff
@@ -0,0 +1,89 @@
+---
+template: main.html
+---
+
+# Notification Tasks
+
+## `notification/slack`
+
+### Overview
+
+The `notification/slack` task posts a slack message about current state the experiment.
+
+### Examples
+
+The following task in the start action of an experiment creates a notification that will be posted to the slack channel with id `channel` during the execution of the `start` action.
```

Comment:
The following task creates a notification (I realize start action is described in this example, but I feel is better to simply describe the task and leave out the action). This task is embeddable into start, loop or finish actions.